### PR TITLE
Swap hawk for beholder

### PR DESCRIPTION
**Pull request by dpassen**

### CODE REVIEW

**Comment by green-coder** [comment]
What are the objective reasons which would justify swapping Hawk for Beholder?

File: lib/processor/deps.edn [hunk 1]
```diff
@@ -5,7 +5,8 @@
            org.clojure/tools.deps.alpha {:mvn/version "0.12.1120"}
            org.clojure/tools.analyzer   {:mvn/version "1.1.0"}
            ;rewrite-clj/rewrite-clj      {:mvn/version "0.6.1"}
-           hawk/hawk                    {:mvn/version "0.2.11"}
+           com.nextjournal/beholder     {:mvn/version "1.0.0"}
+           org.slf4j/slf4j-nop          {:mvn/version "1.7.36"}
```
Comment (green-coder) [inline lib/processor/deps.edn:9]: Why including `org.slf4j/slf4j-nop`? Is it related to `com.nextjournal/beholder`?

**Comment by dpassen** [comment]
hawk has issues with WSL-2 on Windows (https://github.com/wkf/hawk/issues/29) and on M1 Macs (https://github.com/lambdaisland/kaocha/blob/main/CHANGELOG.md#changed-2)

File: lib/processor/deps.edn (hunk 1 above)
Comment (dpassen) [reply lib/processor/deps.edn:9]: Yes. From the README (https://github.com/nextjournal/beholder/blob/main/README.md),
```
The core directory-watcher library is designed to have minimal dependencies; currently it only depends on slf4j-api (for internal logging, which can be disabled by passing a NOPLogger in the builder) and jna (for the macOS watcher implementation).
```

**Comment by green-coder** [comment]
Thank you for the contribution. I will accept and merge the PR once it is out of the Draft state.

**Comment by dpassen** [comment]
Thanks -- I'll try to have it ready tomorrow

**Comment by green-coder** [comment]
Thank you.

Tested on M1, works fine and fast 🎉

**Pull request merged**